api/controller/kubernetes_cluster_api.proto

```diff
@@ -24,6 +24,18 @@ message CheckKubernetesClusterConnectionRequest {
       msg_exists: true
     }
   ];
+  // Version of the PXC operator that is to be checked if it's installed.
+  string pxc_operator_version = 2 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  // Version of the PSMDB operator that is to be checked if it's installed.
+  string psmdb_operator_version = 3 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
```

Comment:
> checked if it's installed

And what if it's not installed?

Reply:
Then status telling it's not istalled is returned. For exemple: CheckKubernetesClusterConnectionResponse->operators->psmdb->status.